app/models/comment.rb

```diff
@@ -1,4 +1,13 @@
 class Comment < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  belongs_to :user
   belongs_to :post
+
+  attribute :text, :text
+
+  after_create :update_posts_counter
+  after_destroy :update_posts_counter
+
+  def update_posts_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
```suggestion
  private
  
  def update_posts_counter
    post.update(comments_counter: post.comments.count)
  end
```
[OPTIONAL]
- [ ] Nice work using one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) to call the `#update_posts_counter` method every time a new comment has been saved or destroy :clap: However it would be nice to make the method private since we only need it in the comment model

- [ ] Please change the method name to something more appropriate to its functionality (**e.g.** `update_comments_counter`)